pkg/resource/db_instance/sdk.go

```diff
@@ -848,21 +848,15 @@ func (rm *resourceManager) sdkCreate(
 	} else {
 		ko.Status.DBParameterGroups = nil
 	}
-	// TODO(jaypipes): Commenting this out until Issue #178 is resolved. The Go
-	// type of the DBSecurityGroups field in the Create input and output shapes
-	// is different, leading to a compilation failure below.
-	//
-	// https://github.com/aws-controllers-k8s/community/issues/178
-	//if resp.DBInstance.DBSecurityGroups != nil {
-	//	f17 := []*string{}
-	//	for _, f17iter := range resp.DBInstance.DBSecurityGroups {
-	//		var f17elem string
-	//		f17 = append(f17, f17elem)
-	//	}
-	//	ko.Spec.DBSecurityGroupNames = f17
-	//} else {
-	//	ko.Spec.DBSecurityGroupNames = nil
-	//}
```

Comment:
I see that all of this was previously disabled? We typically don't edit generated code, but I think Jay did this on purpose (given https://github.com/aws-controllers-k8s/community/issues/178 is still open)

Reply:
Yes, I still had to modify the generated code for `make test` to work properly.

The compilation error is fixed with this change:

```
-			var f17elem string
-			f17 = append(f17, f17elem)
+			f17 = append(f17, f17iter.DBSecurityGroupName)
```

Not sure if we should keep mine, or just comment it all out again like @jaypipes did.

Reply:
I think it's fine to leave it in, but at least add a comment so we know why it is being overridden.

Reply:
Comment added. Thanks @RedbackThomson .

Reply:
Nice catch on this @RedbackThomson :) Ryan, thanks for the fix!